### DISCUSSION TITLE
CMU 开始组建 OSPO，并招聘 OSPO Director

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#80] [日常讨论]
CMU 貌似是继 Hopkins 和 RIT 之后的又一所设立 OSPO 的大学~

News: https://cmu.wd5.myworkdayjobs.com/en-US/CMU/job/Pittsburgh-PA/Director---Open-Source-Program-Office_2018040

The **Open Source Program Office (OSPO)** Director will lead the establishment of an Open Source Programs Office at CMU, and provide leadership for the open source software community across the university.  Like other universities, CMU features diverse contributors and users ranging from single developers to large-scale projects, and from undergraduate students working on small-scale work to major organizational units with significant investments in open source software. The Director therefore will need to build relations at all levels of the university, and blend an appreciation of open source activities and technologies with an understanding of the mission of a research-intensive and highly computational university.
 
Note: This is a grant-funded position; position is fixed term expected duration of three years.
Core responsibilities will include:

- Leadership of the CMU OSPO, including development of strategy
- Develop plans to grow and support the open source community
- Pursue opportunities to secure grants and raise funds to advance the work of the CMU OSPO
- Building an open source community across CMU by building and sustaining relationships with university leaders, faculty, staff and students
- Developing integrations with existing open scholarship programs at CMU, including the OSDC, open data and open science initiatives led by the University Libraries
- Participate in existing open source communities such as OSPO++
- Represent the university in relationships with government and with other universities
- Supervision of the OSPO Program Manager

You must demonstrate:

- Demonstrated results in building, supporting and empowering community in a leadership role
- Excellent oral, written and interpersonal skills
- Ability to operate effectively in complex, dynamic environments
- Ability to work independently while engaging colleagues and supervisors at strategic or key moments
- Flexibility, excellence, and passion are vital qualities within University Libraries.  Inclusion, collaboration and cultural sensitivity are valued competencies at CMU. Therefore, we are in search of a team member who is able to effectively interact with a varied population of internal and external partners at a high level of integrity. We are looking for someone who shares our values and who will support the mission of the university through their work. 

Qualifications: 

- Experience with leadership of an OSPO in a research intensive university
- Experience of leadership of open scholarship programs in a research intensive university library
- Knowledge of open source ecosystem, preferably with evidence of building or establishing relationships
- Demonstrated experience in securing grant funding, ideally from foundations and charitable organizations

Requirements:

- Successful Background Check

**Comment by wangleo61** [comment]
先行者，王老师也搞一个

**Comment by LinuxSuRen** [comment]
王老师也搞一个 +1
